sympy/core/tests/test_function.py

```diff
@@ -977,11 +1038,6 @@ def test_issue_13843():
     assert Derivative(f(x), (x, n)).doit() == Derivative(f(x), (x, n))
 
 
-def test_issue_13873():
-    from sympy.abc import x
-    assert sin(x).diff((x, -1)).is_Derivative
```

Comment:
Why remove this?

Reply:
Negative counts are no longer permitted; it is tested at line 1171 below:

https://github.com/sympy/sympy/blob/03ffa11604bcd59a75e3bb6acebe9e34416f8c65/sympy/core/tests/test_function.py#L1171-L1173

Reply:
Ok, this is fine. Odd that the original test didn't test for an error being that it references the issue.

Reply:
It originally just left them unevaluated. Then when `series` generated negative indices and divided by `factorial(neg)` (which is `oo`) it eliminated the term having a Derivative with a negative count.